delivery.yaml

```diff
@@ -4,7 +4,7 @@ pipeline:
     type: script
     overlay: guild-python/legacy
     env:
-        PYENV_VERSION: 3.6.9
+        PYENV_VERSION: 3.6.10
```

Comment:
line https://github.com/zalando-nakadi/bubuku/pull/148/files#diff-8f20c8ae917a08b05bc008d227a1ad10R28 should it be `bubuku-appliance` vs `bubuku` ?

Reply:
line 28

Reply:
@adyach probably. To be honest I didn't understand why it publishes to identical images. Is it the case that "test" means for validation on PRs and the other is after merging?

Reply:
ef96170

Reply:
i do not know the reason, usually, the image is the same, but configuration is different